server/routes/api.js

```diff
@@ -48,4 +48,74 @@ router.get('/users', (req, res) => {
     });
 });
 
+router.get('/Students', (req, res) => {
+  connection((db) => {
+  db.collection('Students')
+    .find()
+    .toArray()
+    .then((users) => {
+    response.data = users;
+  res.json(response);
+})
+.catch((err) => {
+    sendError(err, res);
+});
+});
+});
+router.get('/Reviews', (req, res) => {
+  connection((db) => {
+  db.collection('Reviews')
+    .find()
+    .toArray()
+    .then((users) => {
+    response.data = users;
+  res.json(response);
+})
+.catch((err) => {
+    sendError(err, res);
+});
+});
+});
+router.get('/Classes', (req, res) => {
+  connection((db) => {
+  db.collection('Classes')
+    .find()
+    .toArray()
+    .then((users) => {
+    response.data = users;
+  res.json(response);
+})
+.catch((err) => {
+    sendError(err, res);
+});
+});
+});
+router.get('/Teachers', (req, res) => {
+  connection((db) => {
+  db.collection('Teachers')
+    .find()
+    .toArray()
+    .then((users) => {
+    response.data = users;
+  res.json(response);
+})
+.catch((err) => {
+    sendError(err, res);
+});
+});
+});
+router.get('/Administors', (req, res) => {
+  connection((db) => {
+  db.collection('Administors')
```

Comment:
Should be `Administrators` here as well, make sure to make this fix on the database end as well.